client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable wpcalypso/i18n-no-variables */
```

Comment:
If we really need to disable a rule, I prefer using an inline comment to make it easier to see what's being disabled.
`// eslint-disable-line your-disabled-rule`